main/lsp/lsp_helpers.cc

```diff
@@ -92,7 +92,11 @@ string LSPLoop::fileRef2Uri(const core::GlobalState &gs, core::FileRef file) {
 unique_ptr<Range> loc2Range(const core::GlobalState &gs, core::Loc loc) {
     unique_ptr<Position> start;
     unique_ptr<Position> end;
-    if (!loc.file().exists()) {
+    if (!loc.exists()) {
+        // if we disable the stdlib (e.g. to speed up testing in fuzzers), then a loc may not exist.
+        start = make_unique<Position>(0, 0);
+        end = make_unique<Position>(0, 0);
+    } else if (!loc.file().exists()) {
```

Comment:
Ah, we had a check! Can you just change `!loc.file().exists()` to `!loc.exists()`? They should be equivalent.

Reply:
Ah, I see. What are those numbers? (1, 1) and (2, 0)?

Reply:
It's the full second line of the code. I think it's so the location maps to *something* plausible. You can change it to (0, 0) and (1, 0), since LSP uses 0-indexed lines and columns.